### DISCUSSION TITLE
DNS远程解析是什么？未在UI上找到相关选项

### DISCUSSION THREAD

**Discussion by zzz6839** [vernesong/OpenClash#2628] [Q&A] [answered]
RT，想问下该如何关闭DNS远程解析，客户端版本号

v0.45.47-beta

![image](https://user-images.githubusercontent.com/57741549/184499026-26c24095-8e3c-44fa-a300-c61c9ac43e79.png)

**Comment by vernesong** [comment, marked as answer]
官方内核的dns选项，redir模式时设置才会出现

**Reply by zzz6839** [reply]
虽然是用的fakeip模式不知道为什么会启用这个也不知道怎么关掉，但是感谢回复。
